src/Dependency/Console/SymfonyConsoleInputJsonOutputAdapter.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\Dependency\Console;
+
+class SymfonyConsoleInputJsonOutputAdapter extends SymfonyConsoleInputOutputAdapter
+{
+    /**
+     * @var string
+     */
+    public const MESSAGE_LIST_KEY = 'message-list';
+
+    /**
+     * @var string
+     */
+    public const WARNING_LIST_KEY = 'warning-list';
+
+    /**
+     * @var array<string>
+     */
+    protected array $successList = [];
+
+    /**
+     * @var array<string>
+     */
+    protected array $warningList = [];
+
+    /**
+     * @param array<string>|string $messages
+     * @param bool $newline
+     * @param int $options
+     *
+     * @return void
+     */
+    public function write($messages, bool $newline = false, int $options = 0): void
+    {
+        $this->successList[] = is_array($messages) ? implode(' ', $messages) : $messages;
+    }
+
+    /**
+     * @param array<string>|string $messages
+     * @param int $options
+     * @param bool $isWarning
+     *
+     * @return void
+     */
+    public function writeln($messages, int $options = 0, bool $isWarning = false): void
+    {
+        $this->successList[] = is_array($messages) ? implode(' ', $messages) : $messages;
```

Comment:
It has the same behavior as write(), so maybe instead of copy/paste just use this method as a wrap:

```
$this->write($messages, $options, $isWarning);
```

Reply:
++